src/main/java/salesforce/ui/pages/classic/contracts/ClassicNewContractPage.java

```diff
@@ -0,0 +1,249 @@
+    private WebElement billingPostalCode;
+    @FindBy(css = "#ctrc25country")
+    private WebElement billingCountry;
+    @FindBy(css = "#SpecialTerms")
+    private WebElement specialTerms;
+    @FindBy(css = "#Description")
+    private WebElement description;
+    @FindBy(css = "td[id='bottomButtonRow'] input[title='Save']")
+    private WebElement save;
+    @FindBy(css = "td[id='bottomButtonRow'] input[title='Save & New']")
+    private WebElement saveAndNew;
+    @FindBy(css = "td[id='bottomButtonRow'] input[title='Cancel']")
+    private WebElement cancel;
+
+    /**
+     * Waits for the page to be loaded.
+     */
+    @Override
+    protected void waitForPageLoaded() {
+        webElementAction.waitForVisible(accountName);
+    }
+
+    /**
+     * Sets the account name on a contract.
+     *
+     * @param newAccountName
+     */
+    public void setAccountName(final String newAccountName) {
+        this.accountName.sendKeys(newAccountName);
+    }
+
+    /**
+     * Sets the CustomerSignedBy element.
+     *
+     * @param newCustomerSignedBy to be typed.
+     */
+    public void setCustomerSignedBy(final String newCustomerSignedBy) {
+        this.customerSignedBy.sendKeys(newCustomerSignedBy);
+    }
+
+    /**
+     * Sets the CustomerSignedTittle element.
+     *
+     * @param newCustomerSignedTittle to be typed.
+     */
+    public void setCustomerSignedTittle(final String newCustomerSignedTittle) {
+        this.customerSignedTittle.sendKeys(newCustomerSignedTittle);
+    }
+
+    /**
+     * Sets the customerSignedDate element.
+     *
+     * @param newCustomerSignedDate to be typed.
+     */
+    public void setCustomerSignedDate(final String newCustomerSignedDate) {
+        this.customerSignedDate.sendKeys(newCustomerSignedDate);
+    }
+
+    /**
+     * Sets the priceBook element.
+     *
+     * @param newPriceBook to be typed.
+     */
+    public void setPriceBook(final String newPriceBook) {
+        if (newPriceBook == "Standandar") {
+            this.priceBook.click();
+            priceBook.findElement(By.xpath("//option[. = 'Standard']")).click();
+        }
+    }
+
+    /**
+     * Sets the contractStartDate element.
+     *
+     * @param newContractStartDate to be typed.
+     */
+    public void setContractStartDate(final String newContractStartDate) {
+        this.contractStartDate.sendKeys(newContractStartDate);
+    }
+
+    /**
+     * Sets the contractTermMonths element.
+     *
+     * @param newContractTermMonths to be typed.
+     */
+    public void setContractTermMonths(final String newContractTermMonths) {
+        this.contractTermMonths.sendKeys(newContractTermMonths);
+    }
+
+    /**
+     * Sets the ownerExpirationNotice element.
+     *
+     * @param newOwnerExpirationNotice to be typed.
+     */
+    public void setOwnerExpirationNotice(final String newOwnerExpirationNotice) {
+        this.ownerExpirationNotice.click();
+        switch (newOwnerExpirationNotice) {
```

Comment:
Instead of the switch you could build the locator, the code would be similar to:
`ownerExpirationNotice.findElement(By.xpath("//option[. = '" + newOwnerExpirationNotice + "']")).click();`